### PR TITLE
Remove unnecessary button styles

### PR DESCRIPTION
**Pull request by calebmeyer**
Atom already includes a quite good button style (see the styleguide), so we can make use of that work and make the fix button fit better into the theme.

## One Dark
image

## One Light
image

## Previous look
image

### CODE REVIEW

File: lib/tooltip/fix-button.tsx [hunk 1]
```diff
@@ -4,8 +4,12 @@ export interface Props {
...
-    <button className="fix-btn" onClick={props.onClick}>
+    <button className="btn fix-btn" onClick={props.onClick}>
       Fix
     </button>
   )
 }
+
+let x;
+
+console.log(x)
```
Comment (aminya) [inline lib/tooltip/fix-button.tsx:15]:
```suggestion
```

File: lib/tooltip/fix-button.tsx [hunk 2]
```diff
@@ -4,8 +4,12 @@ export interface Props {
...
 export function FixButton(props: Props) {
   return (
-    <button className="fix-btn" onClick={props.onClick}>
+    <button className="btn fix-btn" onClick={props.onClick}>
       Fix
     </button>
   )
 }
+
+let x;
```
[the remark this answers is not in the document]
Comment (calebmeyer) [reply lib/tooltip/fix-button.tsx]: https://github.com/calebmeyer/linter-ui-default/commit/b02925aec6e603b3ea8fb2a05fa19dab5fd8bc54

File: lib/tooltip/fix-button.tsx (hunk 1 above)
Comment (calebmeyer) [reply lib/tooltip/fix-button.tsx]: https://github.com/calebmeyer/linter-ui-default/commit/b02925aec6e603b3ea8fb2a05fa19dab5fd8bc54

File: dist/index.js.map [hunk 3]
```diff
@@ -1 +1 @@
...
...sCAFM;AAGZ+C,EAAAA,MAAM,EAAE,4FAHI;AAIZG,EAAAA,EAAE,EAAE,wDAJQ;AAKZF,EAAAA,OAAO,EAAE,sCALG;AAMZG,EAAAA,UAAU,EAAE,yCANA;AAOZC,EAAAA,IAAI,EAAE,uEAPM;AAQZC,EAAAA,IAAI,EAAE,aAAa;AAAb,IACF,4DADE,CAC2D;AAD3D,IAEF,yBAFE,CAEwB;AAFxB,IAGF,+BAHE,CAG8B;AAH9B,IAIF,+BAJE,CAI8B;AAJ9B,IAKF,2CALE,CAK0C;AAL1C,IAMF,gDANE,CAM+C;AAN/C,IAOF,qGAPE,CAOoG;AAPpG,IAQF,qFARE,CAQoF;AARpF,IASF,GAjBQ;AAkBZC,EAAAA,GAAG,EAAE,kFAlBO;AAmBZL,EAAAA,OAAO,EAAEgI,QAnBG;AAoBZzH,EAAAA,KAAK,EAAEyH,QApBK;AAqBZxH,EAAAA,QAAQ,EAAE,qCArBE;AAsBZ;AACA;AACAqI,EAAAA,UAAU,EAAE,gFAxBA;AAyBZ/V,EAAAA,IAAI,EAAE;AAzBM,CAAd;AA4BAyL,KAAK,CAACuK,MAAN,GAAe,gCAAf;AACAvK,KAAK,CAACwK,MAAN,GAAe,8DAAf;AACAxK,KAAK,CAAC8B,GAAN,GAAYyG,IAAI,CAACvI,KAAK,CAAC8B,GAAP,CAAJ,CACT1B,OADS,CACD,OADC,EACQJ,KAAK,CAACuK,MADd,EAETnK,OAFS,CAED,OAFC,EAEQJ,KAAK,CAACwK,MAFd,EAGT5B,QAHS,EAAZ;AAKA5I,KAAK,CAACyK,MAAN,GAAe,uBAAf;AACAzK,KAAK,CAACxgB,IAAN,GAAa,8CAAb;AACAwgB,KAAK,CAACxgB,IAAN,GAAa+oB,IAAI,CAACvI,KAAK,CAACxgB,IAAP,EAAa,IAAb,CAAJ,CACV4gB,OADU,CACF,OADE,EACOJ,KAAK,CAACyK,MADb,EAEV7B,QAFU,EAAb;AAIA5I,KAAK,CAAC4F,aAAN,GAAsB2C,IAAI,CAAC,aAAD,CAAJ,CACnBnI,OADmB,CACX,MADW,EACHJ,KAAK,CAACyK,MADH,EAEnB7B,QAFmB,EAAtB;AAIA5I,KAAK,CAAC4B,IAAN,GAAa2G,IAAI,CAACvI,KAAK,CAA
```
[the remark this answers is not in the document]
Comment (calebmeyer) [reply dist/index.js.map]: Sorry, github desktop just included it, so I thought it wasn't gitignored. I'll get it removed

[the remark this answers is not in the document]
Comment (aminya) [reply dist/index.js.map]: No problem. I should add a git-hook to prevent those from being included.

**Review by aminya — APPROVED** [review]

File: dist/index.js.map (hunk 3 above)
[the remark this answers is not in the document]
Comment (calebmeyer) [reply dist/index.js.map]: https://github.com/calebmeyer/linter-ui-default/commit/8df56b04ae5bbb12af3fd73d0df3399a7437820e

**Comment by aminya** [comment]
Thanks!

**Pull request merged**